client.php

```diff
@@ -33,9 +33,9 @@ function authenticate(){
         );
         $result = curl_post($url, $fields, array(CURLOPT_HEADER => 1));
         $res = http_parse($result);
-        
```

Comment:
My editor automatically removes trailing whitespace. It doesn't hurt to leave it removed, though.